_data/accessibility-tests/footer.yml

```diff
@@ -0,0 +1,97 @@
+title: Footer
+component_status: pass
+test_items:
+# General tests
+  - summary: Some fields have auto-populate capability. 
+    summary_additional: When you navigate to an input field in the footer (like an email address), there will be an option in the browser to auto-populate information.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.5
+  - summary: Footer font meets color contrast requirements.
+    summary_additional: When you use a color contrast checker to view the footer, contrast between the font color and background color is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.3
+  - summary: Footer elements are visible and functional when screen is magnified.
+    summary_additional: When you zoom in to 400%, footer links and elements are stacked and visible. No left-to-right scrolling is required.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.10
+  - summary: Footer icons and non-text content meet color contrast requirements. 
+    summary_additional: When you use a color contrast checker to view the footer, contrast between the icon or other non-text elements and adjacent colors is at least 3:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.11
+  - summary: Links provide direct location description.
+    summary_additional: You can read the text of any link in the footer to understand where the link will take you.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.4
+  - summary: Form field instructions are visible.
```

Comment:
@amycole501 @alex-hull @mahoneycm 
**suggestion (non-blocking):** Recommend naming the applicable variant in the summary.
```suggestion
  - summary: Form field instructions are visible in the big footer variant.
```

Reply:
I wrote up a note in the spreadsheet to this effect, but there are two success criteria inside 1.3.5 that have to be met in order to pass. Ours passes one but not the other. That said, I think we can do as you suggest for now and ensure this is revisited soon and we can discuss it further as it applies to not just the footer but to other personal data input fields.